ds3-sdk/src/main/java/com/spectralogic/ds3client/commands/parsers/GetObjectResponseParser.java

```diff
@@ -63,7 +63,6 @@ protected void download(final long objectSize, final WebResponse response) throw
         try (final InputStream responseStream = response.getResponseStream()) {
             final long startTime = PerformanceUtils.getCurrentTime();
             final long totalBytes = IOUtils.copy(responseStream, destinationChannel, bufferSize, objName, false);
-            destinationChannel.close();
```

Comment:
Ah.  Intellij hides the auto gen comment in the copyright notice.  We can't close the channel in the response parser if we want to be able to resume an interrupted http transfer.  The java sdk has this notion of an AutoCloseableCache that keeps track of channel lifetimes.  Closing the channel here subverts the cache lifetime management.